packages/core/src/awsService/cloudWatchLogs/liveTailLogStreamSubmenu.ts

```diff
@@ -0,0 +1,168 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Prompter, PromptResult } from '../../shared/ui/prompter'
+import { DefaultCloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
+import { createCommonButtons } from '../../shared/ui/buttons'
+import { createInputBox, InputBoxPrompter } from '../../shared/ui/inputPrompter'
+import { createQuickPick, DataQuickPickItem, QuickPickPrompter } from '../../shared/ui/pickerPrompter'
+import { pageableToCollection } from '../../shared/utilities/collectionUtils'
+import { CloudWatchLogs } from 'aws-sdk'
+import { isValidResponse, StepEstimator } from '../../shared/wizards/wizard'
+import { isNonNullable } from '../../shared/utilities/tsUtils'
+
+export enum LogStreamFilterType {
```

Comment:
instead of an enum you can just use a type:
```
export type LogStreamFileType = 'menu' | 'prefix' | 'specific' | 'all'
```
you'll still get autocompletion and everything